Viperinius.Plugin.SpotifyImport/Configuration/playlistConfig.js

```diff
@@ -280,14 +286,23 @@ export default function (view) {
             
             playlists.forEach(pl => {
                 // match a given spotify id with or without a prepended url or uri part
-                const match = /^(https?:\/\/open.spotify.com\/(playlist|user)\/|spotify:playlist:)?([a-zA-Z0-9]+)$/gm.exec(pl.Id);
+                const match = /^(https?:\/\/open.spotify.com\/playlist\/|spotify:playlist:)?([a-zA-Z0-9]+)$/gm.exec(pl.Id);
                 if (match !== null && match.length > 3) {
                     pl.Id = match[3];
```

Comment:
One of the previous regex groups was removed, so the count needs to be decremented:
```suggestion
                if (match !== null && match.length > 2) {
                    pl.Id = match[2];
```